pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -90,9 +92,12 @@ type InstallerController struct {
 
 	installerPodMutationFns []InstallerPodMutationFunc
 
+	startupMonitorDeployed func() (bool, error)
```

Comment:
would it make sense to rename to `startupMonitorEnabledFn` ?

`startupMonitorDeployed` implies the monitor was deployed, whereas we only know it is enabled.

Reply:
fixed